Examples/UIExplorer/NavigationExperimental/NavigationCompositionExample.js

```diff
@@ -178,8 +178,10 @@ class ExampleTabScreen extends React.Component {
   _renderHeader(props: NavigationSceneRendererProps) {
     return (
       <NavigationHeader
-        {...props}
-        getTitle={state => stateTypeTitleMap(state)}
+        navigationProps={props}
+        renderTitleComponent={(navigationProps, scene) => {
```

Comment:
does your change make this prop required? Lets make the prop optional and leave it out of this example.